### PR TITLE
mvcc: add metrics for versions

### PR DESCRIPTION
**Pull request by disksing**
@siddontang @ngaut 

Not sure how much this will affect performance.

### CODE REVIEW

File: src/storage/mvcc/metrics.rs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+use prometheus::{Histogram, exponential_buckets};
+
+lazy_static! {
+    pub static ref MVCC_VERSIONS_HISTOGRAM: Histogram =
+        register_histogram!(
+            histogram_opts!{
+                "tikv_storage_mvcc_versions",
+                "Histogram of versions for each key",
+                [ exponential_buckets(1.0, 2.0, 10).unwrap() ]
```
Comment (siddontang) [inline src/storage/mvcc/metrics.rs]: only [-inf, 1, 2, 10, +inf] is enough?

Comment (disksing) [inline src/storage/mvcc/metrics.rs]: `exponential_buckets` will create `[1, 2, 4, 8, ..., 1024]`.

**Comment by siddontang** [comment]
LGTM

**Comment by ngaut** [comment]
PTAL @BusyJay @zhangjinpeng1987

**Comment by zhangjinpeng1987** [comment]
LGTM

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**